libraries/Microsoft.Bot.Builder.AI.QnA/QnAMakerOptions.cs

```diff
@@ -113,12 +114,35 @@ public QnAMakerOptions()
         public string RankerType { get; set; }
 
         /// <summary>
-        /// Gets or sets <see cref="StrictFilters"/> join operator.
+        /// Gets or sets strictFiltersJoinOperator - join operator on metadata strict filters.
         /// </summary>
-        /// <value>
-        /// A value used for join operation of StrictFilters <see cref="StrictFilters"/>.
-        /// </value>
+        /// <value>StrictFiltersJoinOperator - join operator on metadata strict filters.</value>
```

Comment:
Summary and value can be improved. Please do not use "Gets or sets <property value>".